web/src/services/theme/reearthTheme/common/colors.ts

```diff
@@ -4,6 +4,19 @@ const colors = {
     white: "#FFFFFF",
     transparentBlack: "rgba(0,0,0,0.7)",
     transparentLight: "rgba(0,0,0,0.4)",
+    mainSelect: "#3B3CD0",
+    weakOutline: "#525252",
+    bg: {
+      1: "#1e2086",
+      2: "#df3013",
+      3: "#161616",
+      4: "#393939",
+    },
+    text: {
+      main: "#E0E0E0",
+      weak: "#6F6F6F",
+      weaker: "#525252",
+    },
```

Comment:
It seems like @lavalse is using new colours across the design. Please don't add these colors. Also, common should be COMMON. These colors are very specific to dark theme...

Reply:
as discussed , I removed those colors , and if any one is exists already I replaced with it , also I added them to common assuming that we will have same for light

Reply:
Sorry can you remove those colors too. It doesn't look like you are using any of them in this PR so it would be nice to just leave theme as-is until you make your plan for the color/theme task(s) 🙏🏻

Reply:
Already done